src/DebugEngineHost.VSCode/DebugEngineHost.VSCode.csproj

```diff
@@ -108,8 +108,7 @@
       <DropUnsignedFile Condition="'$(IsMonoRuntime)' == 'false'" Include="$(OutDir)\Microsoft.DebugEngineHost.pdb" />
     </ItemGroup>
   </Target>
-  <Import Condition="'$(IsCoreClr)' == 'true'" Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
-  <Import Condition="'$(IsCoreClr)' == 'false'" Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```

Comment:
This doesn't look right to be. When you installed 2019, did you select ".NET Portable Library targeting pack"?

Reply:
Didn't notice that this was just DebugEngineHost.VSCode.csproj. I am slightly worried that we have PCL projects, and yet we are no longer pulling in the portable targets. But I guess it hasn't hurt us yet.